packages/farms/src/types.ts

```diff
@@ -57,6 +57,7 @@ export interface FarmConfigBaseProps {
     aptIncentiveInfo: number
   }
   boosted?: boolean
+  showPoolEarnAptTooltips?: boolean
```

Comment:
can we not adding extra type to the farms? The reward feels like a extra layer of reward on top of farm to me. How about we follow what we did with merkl pool here?
https://github.com/pancakeswap/pancake-frontend/blob/develop/apps/web/src/config/constants/merklPools.json

Similarly, we can have a apt rewarded farm config in aptos web